packages/lesswrong/server/scripts/migrateCollections.ts

```diff
@@ -38,6 +38,10 @@ const formatters: Partial<Record<CollectionNameString, (document: DbObject) => D
     migration.succeeded ??= false;
     return migration;
   },
+  Users: (user: DbUser): DbUser => {
+    user.isAdmin = !!user.isAdmin;
```

Comment:
Is that really all the nasty hacks? I would have expected worse.

Reply:
Yep, and it was only one record. I could've just fixed it in the DB, but I figured I'd leave it here for LW too.